RPT_Application/src/main/java/com/harman/report_database/ReportService.java

```diff
@@ -0,0 +1,54 @@
+package com.harman.report_database;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.harman.user_database.Employee;
+import com.harman.user_database.EmployeeService;
+
+@Service
+public class ReportService {
+
+	@Autowired
+	private ReportRepository reportRepository;
+
+	private static final Logger LOGGER = Logger.getLogger(EmployeeService.class.getName());
+
+	public List<Report> findAll() {
+		return reportRepository.findAll();
+	}
+
+	public void save(Report report) {
+		if (report == null) {
+			LOGGER.log(Level.SEVERE, "Report is null. And that's bad. Check why do you save null Report.");
+			return;
+		}
+		reportRepository.save(report);
+	}
+
+	public Report findById(Long id) {
+		if (reportRepository.findById(id).isPresent()) {
+			return reportRepository.findById(id).get();
+		} else
+			return new Report();
```

Comment:
1. Braces should be used.
2. Why returning empty Report if there is no report in the database?